### PR TITLE
MAT-4369 update cqframework version to newest

### PR DESCRIPTION
**Pull request by sb-cecilialiu**
## CQL to ELM Translation Service PR

Jira Ticket: [MAT-4369](https://jira.cms.gov/browse/MAT-4369)
(Optional) Related Tickets:

### All Submissions
* [x] This PR has the JIRA linked.
* [ ] Required tests are included.
* [x] No extemporaneous files are included (i.e Complied files or testing results).
* [x] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [x] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
   <description>Translates CQL to ELM for MADiE</description>
 
   <properties>
-    <cqframework.version>2.4.0</cqframework.version>
+    <cqframework.version>2.5.0</cqframework.version>
```
Comment (adongare) [inline pom.xml]: I think v2.6.0 is available. we can use it if 2.5.0 is not a strict requirement.

Comment (sb-cecilialiu) [reply pom.xml]: @adongare Thanks a lot Ashok! I changed it to 2.6.0.

**Review by adongare — APPROVED** [review]
